src/components/DetailComponent/DetailComponent.js

```diff
@@ -0,0 +1,253 @@
+import { Container, Grid } from "@material-ui/core";
+import React, { Fragment, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { IMAGE_PATH } from "../../store/constants";
+import Typography from "@mui/material/Typography";
```

Comment:
Absolute imports should be imported above.

Reply:
fixed